### DISCUSSION TITLE
There's no "Report" button/link for me while on Desktop/Mobile - it only exists on iOS or through the documentation for my user

### DISCUSSION THREAD

**Discussion by chris-c-thomas** [community/community#46678] [Discussions]
### Select Topic Area

Show & Tell

### Body

@Akash1134 this isn't an answer as far as I can tell. There are no abuse reporting tools available to me that I can find. A dropdown that at most let's me copy the link.

Your info is just a bulleted list. I can't report anything with that. There's so much spam on these discussion boards at times. 

<img width="1307" alt="Screenshot 2023-02-07 at 3 46 29 PM" src="https://user-images.githubusercontent.com/22353987/217374966-7bd25ebb-b6c6-4132-85ba-b4dc85bf4fe5.png">

I'm sorry if I'm missing something but I really can't find a report abuse button or anything, anywhere from my side of things.

**Comment by chris-c-thomas** [comment]
Here is an example of what I mean @Akash1134

On the iOS App there's a report button. But on in the browser on Desktop or Mobile I either don't seem to have it or I can't find it.

**Here on iOS:**

![image](https://user-images.githubusercontent.com/22353987/217998425-9c007581-612d-414f-9ac7-313bbcd8d460.jpeg)

---

**Not here on Desktop / Mobile Browser:**

![image](https://user-images.githubusercontent.com/22353987/217998435-fd6295db-5248-48fb-986f-43554b57d85f.jpeg)


I would think they'd be in similar spots?

**Comment by chris-c-thomas** [comment]
Apologies, I uploaded the wrong images the first time if anyone saw that.

**Comment by Akash1134** [comment]
Hi @chris-c-thomas Sorry about the confusion caused. You need to click on "Report Content" and then follow the process mentioned in this [documentation](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#about-reporting-abuse-or-spam). Let me know if you have any further queries.

**Comment by chris-c-thomas** [comment]
@Akash1134 - On Desktop/Mobile, there is **NO** "Report" button in the discussions area! **NONE**!

I can use the link you provided to support but there is **NO** "Report" button when using GitHub from a computer. At least, not from my user account's point of view. Only on the iOS app is the report button there in discussions. Look at my screenshot below. There is no "Report"

<img width="934" alt="Screenshot 2023-02-10 at 6 17 48 AM" src="https://user-images.githubusercontent.com/22353987/218090707-0d1518c2-ef09-4f12-adc4-cbd222f38196.png">


See? No "Report" button

**Comment by chris-c-thomas** [comment]
It might be there for other users but it's not for me. Only on the iOS app.

**Comment by chris-c-thomas** [comment]
I can go to your user profile and block or report. But I can't report the specific comment/post from within the discussion.

**Comment by Akash1134** [comment]
That's strange @chris-c-thomas. Could you please log out from your account, close your web browser, and try again?

**Comment by Akash1134** [comment]
@chris-c-thomas Something I just noticed, you are using the special repo for community/community. You should be logged in to https://github.com/orgs/community/discussions and using Discussions. If that does not help, please let me know. Thanks for your patience 🙂

**Comment by chris-c-thomas** [comment]
@Akash1134 thank you for working through this with me. I logged out and back in and also went to the non-special org version (in the URL in the photo below) but still do not see the "Report" option. I've tried both Firefox and Chrome browsers.

<img width="1137" alt="Screenshot 2023-02-10 at 7 08 12 AM" src="https://user-images.githubusercontent.com/22353987/218100296-ae971b61-7293-4c5b-bb72-347ab7955e0d.png">

**Reply by Akash1134** [reply]
Hi @chris-c-thomas Thanks for your persistent follow-ups and continued support ✨. I can confirm that this is a discrepancy between the mobile and the desktop platform of Discussions. We have identified this as a bug and our team is working on it.

**Reply by chris-c-thomas** [reply]
This is great to hear @Akash1134! Thank you very much for continuing to help with this. The discussions feature is getting better and better because of it.

**Comment by rootkea** [comment]
Has this been fixed? I also don't see any option to report a spam comment made on a Github issue.
I'm using Github.com on Desktop.
<img width="897" alt="Screenshot 2023-07-27 at 4 55 24 AM" src="https://github.com/community/community/assets/2694559/48c4fd3c-c074-4805-9484-f83453be3c58">

**Comment by thrashwerk** [comment]
Same here. Has anyone looked into this issue?

Edit: The "Report content" is available in this specific discussion on all of the comments but not in other repos' discussions/PRs/issues.

**Reply by bbsixzz** [reply]
Same problem here Safari Version 17.2.1 (19617.1.17.11.12) macOS Sonoma 14.2.1 (23C71)

**Reply by steezeburger** [reply]
Happening to me now too, but only for a specific repo. I can't report this spam/scam comment as there is no "Report content" button

https://github.com/princelai/NiMingAssistant/issues/1#issuecomment-1953404757

**Comment by mercurytoxic** [comment]
I have exactly the same problem and example of an issue that I can't report is [https://github.com/tronprotocol/documentation-en/issues/203](https://github.com/tronprotocol/documentation-en/issues/203) Even though that issue is blantaly spam. Work around is to report the user but that's inconvenient if its a whole thread full of spam.

**Reply by LiteBrite82** [reply]
WOW! That issue list is full of spam! The repo owner should go into settings -> Moderation Options -> Interaction Limits, and adjust some settings. 

for sure, its annoying the report button isn't there, but you should still [report the user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).

**Reply by LiteBrite82** [reply]
The ORG owners and assigned moderators should be able to get the three dot menu report button.

**Reply by codespearhead** [reply]
Should this [1] be updated then?

[1] https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request

I notice spam even in official GitHub repositories [2].

[2] https://github.com/github/advisory-database/issues/4602

It makes more sense for the "Report content" option to be public instead of limited to ORG owners and assigned moderators in my opinion.

**Reply by ycamargo** [reply]
I just tried to report a "spam/phishing" comment that appeared on an issue thread today (and it looks like the same comment is appearing on many other issues posted by different accounts as reported [here](https://github.com/orgs/community/discussions/136842)). And the button is not there on the web version. I was able to report it through the Android mobile app, though.

**Reply by LiteBrite82** [reply]
@ycamargo 
Thanks for the heads up! I flagged the post you linked for review.

**Comment by AdreKiseque** [comment]
I'm having the same issue. I found some spam on the [VS Code discussions repo](https://github.com/microsoft/vscode-discussions/discussions) and there's no option to report it. Very disturbing.
![image](https://github.com/user-attachments/assets/13f00531-ec44-49a4-90c5-36b13706dc7b)

**Comment by AdreKiseque** [comment]
I noticed this thread was just marked closed as resolved? But there aren't any updates on here as far as I can tell and the bug still seems to be present. Was this a mistake?

**Reply by wally-crunk** [reply]
Still a problem, doesn't seem to be a report button on desktop.  Isn't that a problem?

**Comment by BasixKOR** [comment]
I visited GitHub Support Portal to work around this issue and clicked "abuse" as a ticket type and it redirected me to [this article](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request), asking me to report a issue by using a nonexistant button.

Edit: GitHub only shows this option for collaborators.

> In order to report content in an organization repository there are some criteria that first need to be met, e.g. be a collaborator or organization member first. This is intended to prevent false reports or report-bombing on organizations, among other reasons.  It seems in this case these rules prevented you from reporting spammy content, sorry about that.

**Reply by Bjk8kds** [reply]
I also experienced similar and I think it's quite odd.
But thanks anyway.

**Comment by kxtzownsu** [comment]
I can confirm this issue still exists. I tried to report a GitHub Issue comment for doxxing/harassment of someone else, but the report button was missing.

One thing to note is that the same iOS/desktop discrepancy that OP mentioned is also here, so I am able to report it on iOS but it's just a few extra hoops for no reason.

Desktop (github.com): 
<img width="243" height="173" alt="image" src="https://github.com/user-attachments/assets/6abb6ae8-4c73-4a0c-9134-4ab178168b5e" />
(P.S: please ignore the blue box, it's related to my screenshot bind)

iOS App:
<img width="795" height="568" alt="image" src="https://github.com/user-attachments/assets/52491f04-5471-485d-a6ee-906e7f781aaf" />
